cmd/nvidia-ctk/cdi/list/list.go

```diff
@@ -64,6 +64,7 @@ func (m command) build() *cli.Command {
 			Usage:       "specify the directories to scan for CDI specifications",
 			Value:       cli.NewStringSlice(cdi.DefaultSpecDirs...),
 			Destination: &cfg.cdiSpecDirs,
+			EnvVars:     []string{"NVIDIA_CTK_CDI_LIST_SPEC_DIR"},
```

Comment:
How about:
```suggestion
			EnvVars:     []string{"NVIDIA_CTK_CDI_SPEC_DIRS"},
```
Since we would expect them to be globally consistent. Not a blocker though.